trytond/trytond/ir/configuration.py

```diff
@@ -12,6 +13,13 @@ class Configuration(ModelSingleton, ModelSQL):
     hostname = fields.Char("Hostname", strip=False)
     _get_language_cache = Cache('ir_configuration.get_language')
 
+    @classmethod
+    def __register__(cls, module_name):
+        cursor = Transaction().connection.cursor()
```

Comment:
Comment? in `__init__.py` to make sure it stays the first class to be registered ?